src/tests/fixtures/mock_imports.py

```diff
@@ -0,0 +1,7 @@
+import mock
```

Comment:
Is this file still needed? I think with the correct Python version (like the one installed on our pytest image) we should be able to import these modules without issue.

Reply:
Its not needed, but its a convenience when I am working on my laptop at home.  Arguably I could be running the tests in a container on my laptop at home